tests/lib/rules/prefer-const.js

```diff
@@ -117,17 +117,45 @@ ruleTester.run("prefer-const", rule, {
             parser: fixtureParser("babel-eslint5/destructuring-object-spread")
         },
 
-        // https://github.com/eslint/eslint/issues/8308
+        // // https://github.com/eslint/eslint/issues/8308
```

Comment:
I assume this change is unintentional?

Reply:
Oops, nope, mass commenting/uncommenting got me. I'll fix that.